### PR TITLE
Feature/5 dev first version of charts

### PR DESCRIPTION
**Pull request by wayangalihpratama**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207885014926613

### CODE REVIEW

File: src/components/__tests__/Bar.test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Bar from '../Bar';
+import renderer from 'react-test-renderer';
+
+test('renders Bar component', () => {
+  const data = [
```
Comment (dedenbangkit) [inline src/components/__tests__/Bar.test.js:7]: Please add this in the Readme

File: src/components/__tests__/Bar.test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    { label: 'A', value: 10 },
+    { label: 'B', value: 20 },
+    { label: 'C', value: 30 }
+  ];
+
+  const config = {
```
Comment (dedenbangkit) [inline src/components/__tests__/Bar.test.js:13]: also this part

**Review by dedenbangkit — CHANGES_REQUESTED** [review]
Where is the Doughnut.js? Doughnut can also inherit from Bar.js

**Comment by wayangalihpratama** [comment]
> Where is the Doughnut.js? Doughnut can also inherit from Bar.js

Doughnut chart is in Pie.js, should we separate Pie and Doughnut?

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**